doc/usingchapel/privatebugs.rst

```diff
@@ -0,0 +1,36 @@
+.. _readme-privatebugs:
+
+===============================
+Reporting Private Chapel Issues
+===============================
+
+Private bug reports can be mailed directly to the Chapel team at Cray Inc.
+through :disguise:`chapel_bugs@cray.com`.
```

Comment:
Can this be emboldened to make it stand out a bit?
Optional, but I sometimes leave periods off of sentences like this to avoid needing to be careful about not copying and pasting the final period (and I don't think it hurts readability even though it is technically a complete sentence).